strangelove-ventures/sommelier
# feat: handle dynamic base asset

## Description

Before:
Our codebase can only handle 2 base assets, either `USDC` or `ETH`. and that's still hard coded, so it's hard to introduce new base asset.

After:
This PR enable our codebase to handle dynamic base asset, using subgraph data. 

## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- [ ] Check the calculation, is the result correct or not
- [ ] Compare this pr preview, with staging site
- [ ] Check other file that contain calculation in case there's still hard coded value that I miss


You can see the video preview, in slack channel. this video compare staging site with the refactor result.
